components/svg/download_icon.jsx

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React from 'react';
+
+export default class DownloadIcon extends React.PureComponent {
+    render() {
+        return (
+            <span {...this.props}>
+                <svg
```

Comment:
why is this a component, when the rest are svgs?

Reply:
With how the other SVGs are imported, I think it's fine to keep them as SVG files instead of components since that'll likely make it easier if they need to be changed in the future. I forgot that was even an option at the time